src/status_im/components/styles.cljs

```diff
@@ -9,7 +9,7 @@
 (def color-gray2 "#8f838c93")
 (def color-gray3 "#00000040")
 (def color-gray4 "#939ba1")
-(def color-gray5 "#d9dae1")
+(def color-gray5 "#d9dae188")
```

Comment:
I think it's better to use opacity on an object not in color because it's important what is under this object and this color also using with 1 opacity